src/components/InputsTable.js

```diff
@@ -119,12 +126,25 @@ const InputsTable = props => {
           setStructBuilderVisible(false)
         }
       }),
+      (_.some(row => _.has(row.variable, dataTableAttributes), inputTableData)) && h(div,
+        { style: { height: 0.08 * height, width, display: 'flex', alignItems: 'center' } },
+        [h(Link,
+          {
+            style: { marginLeft: 'auto' },
+            onClick: () => _.flow(
+              _.filter(row => _.has(row.variable, dataTableAttributes)),
+              _.forEach(row => setConfiguredInputDefinition(_.set(`[${row.configurationIndex}].source`, { type: 'record_lookup', record_attribute: row.variable })))
+            )(inputTableData)
+          },
+          ['Set all from data table']
```

Comment:
"all" is a slight misnomer. Can we maybe count how many we're proposing to auto-fill and say `Set (5) from data table`? (and maybe with a tooltip telling us exactly which 5 we found?)

Reply:
<img width="406" alt="Screenshot 2023-06-02 at 11 04 52 AM" src="https://github.com/DataBiosphere/cbas-ui/assets/67511512/13f9eb1b-3251-4227-80fc-de3cbed35b6c">

Something to note, this tooltip does have a somewhat strange output since some variables are part of different tasks but have the same name

Reply:
Two thoughts on the tooltip:
- I think if there are lot of inputs whose attribute values can be auto-filled, this tooltip would get verbose and long. I like the idea of tooltip but I am not sure how to best display the information. I am not sure if its possible, but can a tooltip have scroll bar?
- I think we should display the entire input name including the Task name so that variables with same names can be displayed. For example `ExomeGermlineSingleSample. sample_and_unmapped_bams`

Reply:
<img width="291" alt="Screenshot 2023-06-06 at 6 26 26 PM" src="https://github.com/DataBiosphere/cbas-ui/assets/67511512/a95f8db6-30e7-4839-bc6c-c7d3d52b9b2a">

Minor styling update